lib/nylas/draft.rb

```diff
@@ -29,14 +29,34 @@ class Draft
     attribute :unread, :boolean
 
     has_n_of_attribute :events, :event
-    has_n_of_attribute :files, :file
+    has_n_of_attribute :files, :file, read_only: true
+    has_n_of_attribute :file_ids, :string
     attribute :folder, :folder
     has_n_of_attribute :labels, :label
 
     attribute :tracking, :message_tracking
 
     transfer :api, to: %i[events files folder labels]
 
+    def create
+      extract_file_ids unless files.nil? || files.empty?
+      super
+    end
+
+    def save_call
```

Comment:
This can be private I think

Reply:
Yup, tested and confirmed. Change is made.